app/assets/javascripts/discourse/app/mixins/upload-debugging.js

```diff
@@ -19,44 +25,65 @@ export default Mixin.create({
   },
 
   _performanceApiSupport() {
-    performance.mark("testing support 1");
-    performance.mark("testing support 2");
-    const perfMeasure = performance.measure(
+    this._performanceMark("testing support 1");
+    this._performanceMark("testing support 2");
+    const perfMeasure = this._performanceMeasure(
       "performance api support",
       "testing support 1",
       "testing support 2"
     );
     return perfMeasure;
   },
 
+  _performanceMark(markName) {
+    return performance.mark(markName);
+  },
+
+  _performanceMeasure(measureName, startMark, endMark) {
+    let measureResult;
+    try {
+      measureResult = performance.measure(measureName, startMark, endMark);
+    } catch (error) {
+      if (
+        error.message.includes("Failed to execute 'measure' on 'Performance'")
```

Comment:
Is this stable enough to match on? I imagine it might differ based on browser version, locale, etc.?

Reply:
Probably not that great, but it's also not a big deal if we don't log this, it's a nice-to-have